sale_stock_operating_unit/models/stock_move.py

```diff
@@ -12,7 +12,7 @@ def _get_new_picking_values(self):
         """
         Override to add Operating Units to Picking.
         """
-        values = super(StockMove, self)._get_new_picking_values()
+        values = super()._get_new_picking_values()
```

Comment:
in my opinion, it's not necessary to override this function. we have a constraint to make sure the order and its warehouse  having the same value of `operating_unit_id`. So, just let the field `stock_picking.operating_unit_id` computes the value itself.

Reply:
The operating unit is not mandatory on the warehouse and then you could have a shared warehouse and having pickings from several operating units. In that case, the compute will not work.